dotcom-rendering/src/components/marketing/banners/designableBanner/DesignableBanner.tsx

```diff
@@ -345,11 +345,14 @@ const DesignableBanner: ReactComponent<BannerRenderProps> = ({
 						templateSettings.tickerStylingSettings && (
 							<Ticker
 								currencySymbol={tickerSettings.currencySymbol}
-								copy={tickerSettings.copy}
+								copy={{
+									headline: tickerSettings.copy.countLabel,
```

Comment:
In the Ticker component we've renamed `countLabel` to `headline`, though we still have the old name in the data here

Reply:
worth having a backlog ticket to update it everywhere?